rfcs/text/0012-orchestrator-field-set.md

```diff
@@ -19,9 +19,67 @@ add a new schema rather than change existing material.
 
 ## Fields
 
-<!--
-Stage 1: Describe at a high level how this change affects fields. Which fieldsets will be impacted? How many fields overall? Are we primarily adding fields, removing fields, or changing existing fields? The goal here is to understand the fundamental technical implications and likely extent of these changes. ~2-5 sentences.
--->
+The proposed change adds six fields, as described below:
+
+```
+---
+- name: orchestrator
+  title: Orchestrator
+  group: 2
+  short: Fields relevant to container orchestrators.
+  description: >
+    Fields that describe the resources which container orchestrators manage or
+    act upon.
+  type: group
+  fields:
+    - name: cluster
+      level: extended
+      type: object
```

Comment:
I like the flexibility we have here for setting `orchestrator.cluster` as an object. Do you think we should also define several common fields under cluster here? For example `orchestrator.cluster.name`? cc: @jsoriano @ChrsMark

Reply:
Yes, I'd say that the most relevant ones should be defined, at least `cluster.name` and `cluster.url`.

Reply:
👋 Thanks, I've added the `cluster.name` and `cluster.url` fields. I can't think of anything more that would be essential here at the moment, but happy to add more fields if there are any suggestions.

Reply:
++

Reply:
@ferozsalam Thank you!!